src/main/java/org/apache/maven/shared/utils/cli/CommandLineTimeOutException.java

```diff
@@ -41,4 +40,11 @@ public CommandLineTimeOutException( String message, Throwable cause )
         super( message, cause );
     }
 
+    /**
+     * @param message The message of the exception.
+     */
+    public CommandLineTimeOutException( String message )
```

Comment:
`Timeout` according to https://www.oxfordlearnersdictionaries.com/definition/english/timeout?q=timeout

Reply:
Existing exception name ... I can rename .. but will be break, maybe ok for 4.x 😄

Reply:
No, leave it